nodes/scene/objects_mk3.py

```diff
@@ -34,7 +34,7 @@ class SvOB3BDataCollection(bpy.types.PropertyGroup):
 
 class SvOB3BNamesList(bpy.types.UIList):
 
-    def draw_item(self, context, layout, data, item, icon, active_data, active_propname):
+    def draw_item(self, context, layout, data, item, icon, active_data, active_propname, index):
```

Comment:
oh.. i think that variable _is_needed, but wasn't being used correctly without the `index` parameter